packages/squiggle-lang/__tests__/Distributions/Samples_test.res

```diff
@@ -0,0 +1,46 @@
+open Jest
+open Expect
+
+let makeTest = (~only=false, str, item1, item2) =>
+  only
+    ? Only.test(str, () => expect(item1) -> toEqual(item2))
+    : test(str, () => expect(item1) -> toEqual(item2))
+
+describe("Continuous and discrete splits", () => {
+  makeTest(
+    "check splits one",
```

Comment:
splits one? It seems like all wounded up in one bucket. Maybe rename

Reply:
"splits (1)" and "splits (2)" then